arxiv/release/README.md

```diff
@@ -1,3 +1,22 @@
+# How to make a new version arxiv-base
+To make a new version of arxiv-base that gets uploaded to PyPI, commit
+a tag that is a valid python version in git and travis should upload
+the package to PyPI. Other repos can be set up in a similar manner.
```

Comment:
The above makes it sound like the tag commit triggers an automatic upload to PyPi. The documentation below looks like the manual steps I'm used to following when uploading to PyPi.

Reply:
I think I'm going to get rid of most of this stuff. I wanted to automate it and it was working but then it didn't get used.